easy_my_coop_eater/__manifest__.py

```diff
@@ -20,16 +19,14 @@
 ##############################################################################
```

Comment:
could you comply with the new copyright header ? thanks

Reply:
done